src/material.h

```diff
@@ -29,6 +30,9 @@ class material_type
         int _fire_resist;
         int _chip_resist;       // Resistance to physical damage of the item itself
         int _density;   // relative to "powder", which is 1
+        bool _edible;
+
+        mutable std::map<vitamin_id, double> _vitamins;
```

Comment:
Also, putting vitamins in materials is another big warning that something isn't done right.

Materials aren't good for nutritional details.